test/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/SatelliteDynamics.test.cpp

```diff
@@ -803,14 +810,37 @@ TEST (OpenSpaceToolkit_Astrodynamics_Flight_System_Dynamics_SatelliteDynamics, g
         const Vector3d startVelocityCoordinates = (startState.getVelocity()).inUnit(Velocity::Unit::MeterPerSecond).accessCoordinates() ;
         startStateVector[0] = startPositionCoordinates[0]; startStateVector[1] = startPositionCoordinates[1]; startStateVector[2] = startPositionCoordinates[2] ;
         startStateVector[3] = startVelocityCoordinates[0]; startStateVector[4] = startVelocityCoordinates[1]; startStateVector[5] = startVelocityCoordinates[2] ;
+        
+        // Check initial conditions precision
+        EXPECT_GT(1e-15, startStateVector[0] - 6878137.0) ;
+        EXPECT_GT(1e-15, startStateVector[1] - 0.0 ) ;
+        EXPECT_GT(1e-15, startStateVector[2] - 0.0 ) ;
+        EXPECT_GT(1e-15, startStateVector[3] - 0.0 ) ;
+        EXPECT_GT(1e-15, startStateVector[4] - 7612.608170359118) ;
+        EXPECT_GT(1e-15, startStateVector[5] - 0.0 ) ;
```

Comment:
```suggestion
        EXPECT_NEAR(startStateVector[0], 6878137.0, 1e-15) ;
        EXPECT_NEAR(startStateVector[1], 0.0, 1e-15) ;
        EXPECT_NEAR(startStateVector[2], 0.0, 1e-15) ;
        EXPECT_NEAR(startStateVector[3], 0.0, 1e-15) ;
        EXPECT_NEAR(startStateVector[4], 7612.608170359118, 1e-15) ;
        EXPECT_NEAR(startStateVector[5], 0.0, 1e-15) ;
```

Reply:
Haven't tested this but pretty sure it should work

Reply:
Nice, I did a copy-pasta here but I like this. More readable